extras/label_configs/carrots

```diff
@@ -1,5 +1,5 @@
 # Remove Github defaults
--import:clear-defaults
+-delete:all
```

Comment:
well my thought is updating repos with the wrong tags - that was the reason of adding `-delete:all` originally